client/consensus/slots/src/lib.rs

```diff
@@ -249,6 +262,11 @@ pub trait SimpleSlotWorker<B: BlockT> {
 			Some(claim) => claim,
 		};
 
+		if self.should_backoff(slot_number, &chain_head) {
+			info!("Backing off claiming new slot for block authorship.");
```

Comment:
```suggestion
			info!(
				target: self.logging_target(),
				"Backing off claiming new slot for block authorship.",
			);
```

Reply:
In the end it would be much nicer to have this message in the specific implementation, as we could give a real reason why we backed off.
Here we don't know anything, because the strategy could be anything.

Reply:
I think the reason I initially didn't do that was not having easy access to the logging target. But yeah you're right, I'll move it inside